tests/data/dual_labeled.yaml

```diff
@@ -0,0 +1,95 @@
+# Any copyright is dedicated to the Public Domain.
+# https://creativecommons.org/publicdomain/zero/1.0/
+
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+test.dual_labeled:
+  static_static:
+    type: dual_labeled_counter
+    description: >
+      A dual labeled counter with static keys and categories
+    dual_labels:
+      key:
+        description: >
+          The key for the dual labeled counter
+        labels:
+          - key1
+          - key2
+      category:
+        description: >
+          The category for the dual labeled counter
+        labels:
+          - category1
+          - category2
+    bugs:
+      - https://bugzilla.mozilla.org/11137353
+    data_reviews:
+      - http://example.com/reviews
+    notification_emails:
+      - CHANGE-ME@example.com
+    expires: 2100-01-01
+  dynamic_static:
+    type: dual_labeled_counter
+    description: >
+      A dual labeled counter with static keys and dynamic categories
+    dual_labels:
+      key:
+        description: >
+          The key for the dual labeled counter
+        labels:
+          - key1
+          - key2
+      category:
+        description: >
+          The category for the dual labeled counter
+        labels: []
```

Comment:
Empty list? I think it's more likely that folks'll leave the list absent to trigger dynamic labels.

(( I wonder if we should forbid empty lists of labels. A valid interpretation would be "this is statically labeled and so we expect all labels to end up in `__other__`, which is likely a mistake ))